Lib/test/test_peg_generator/test_c_parser.py

```diff
@@ -8,6 +9,10 @@
 from test.support import os_helper
 from test.support.script_helper import assert_python_ok
 
+_pyflags_nodist = sysconfig.get_config_var('PY_CFLAGS_NODIST') or ()
```

Comment:
Shouldn't this be a string?

```suggestion
_pyflags_nodist = sysconfig.get_config_var('PY_CFLAGS_NODIST') or ""
```

Reply:
Yep, it works too :-)